packages/linea-state-verifier/contracts/EVMFetchTarget.sol

```diff
@@ -27,11 +27,13 @@ abstract contract EVMFetchTarget {
             bytes32[] memory commands,
             bytes[] memory constants,
             bytes4 callback,
-            uint256 l2BlockRangeAccepted
+            bytes memory callbackData
         ) = abi.decode(
                 extradata,
-                (IEVMVerifier, address, bytes32[], bytes[], bytes4, uint256)
+                (IEVMVerifier, address, bytes32[], bytes[], bytes4, bytes)
             );
+
+        uint256 l2BlockRangeAccepted = uint256(bytes32(callbackData));
```

Comment:
I have tested that this will take the first 32 bytes (0x..) if the length of callbackData is > 32

Are there scenarios where this value could be greater than 32 bytes and the rightmost data is what is needed?

The name `l2BlockRangeAccepted` feels like a bool - is this accurate?

Reply:
No there should be no such scenarios, the first 32 bytes have to be taken by a uint256 containing the l2BlockRange.
This is not a bool this a number of L2 blocks we can go back in the past starting from the `currentL2BlockNumber` returned by the L1 Rollup contract.

Reply:
makes sense, thanks. 

The name for me of `acceptedL2BlockRangeLength` or `acceptedL2BlockRangeDifference` would be better, but not critical.

Reply:
I would leave an /// @dev Only the first 32 bytes are used for callbackData and is never longer than 32 bytes.

Reply:
Thanks for the feeback I'll change the name and add the @dev comment